helm_deploy/court-case-matcher/values.yml

```diff
@@ -0,0 +1,84 @@
+generic-service:
+  nameOverride: court-case-matcher
+
+  image:
+    repository: quay.io/hmpps/court-case-matcher
+    pullPolicy: Always
+    tag: app_version # override at deployment time
+    port: 8080
+
+  ingress:
+    enabled: true
+    modsecurity_enabled: false
+    hosts:
+      - app-hostname.local    # override per environment
+    path: /
+    annotations:
+      nginx.ingress.kubernetes.io/configuration-snippet: |
+        server_tokens off;
+        location /queue-admin/retry-all-dlqs {
+          deny all;
+          return 401;
+        }
+        location /replay404Hearings {
+          deny all;
+          return 401;
+        }
+
+  allowlist:
+    groups:
+      - internal
+
+  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
+  # namespace_secrets:
+  #   [name of kubernetes secret]:
+  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
+
+  namespace_secrets:
+    crime-portal-gateway-s3-credentials:
+      CRIME_PORTAL_GATEWAY_S3_BUCKET: bucket_name
+    court-case-matcher-secrets:
+      OFFENDER_SEARCH_CLIENT_ID: nomis-oauth-client-id
+      PERSON_RECORD_SEARCH_CLIENT_ID: nomis-oauth-client-id
+      PERSON_RECORD_SEARCH_CLIENT_SECRET: nomis-oauth-client-secret
+      APPINSIGHTS_INSTRUMENTATIONKEY: APPINSIGHTS_INSTRUMENTATIONKEY
+      OFFENDER_SEARCH_CLIENT_SECRET: nomis-oauth-client-secret
+    court-case-matcher-queue-credentials:
+      AWS_SQS_COURT_CASE_MATCHER_QUEUE_NAME: sqs_name
+      AWS_SQS_COURT_CASE_MATCHER_ENDPOINT_URL: sqs_id
+    court-case-matcher-queue-dead-letter-queue-credentials:
+      AWS_SQS_COURT_CASE_MATCHER_DLQ_QUEUE_NAME: sqs_name
+      AWS_SQS_COURT_CASE_MATCHER_DLQ_ENDPOINT_URL: sqs_id
+
+  autoscaling:
+    enabled: true
+    targetCPUUtilizationPercentage: 100
+
+  env:
+    JAVA_OPTS: "-Xmx512m"
+    SPRING_PROFILES_ACTIVE: "instrumented"
+    SERVER_PORT: "8080"
+    OFFENDER_SEARCH_USE_DOB_WITH_PNC: "true"
+
+  retryDlqCronjob:
+    enabled: true
+    # every 10 minutes
+    retryDlqSchedule: "*/10 * * * *"
```

Comment:
This is the default value

Reply:
do you mean the retryDlqSchedule? for 10 minutes 

i had a look at the [generic-service readme](https://github.com/ministryofjustice/hmpps-helm-charts/tree/main/charts/generic-service#retrying-messages-on-a-dead-letter-queue) and it seems like we need to enable it explicitly